### PR TITLE
feat: replace IsCreated with Exists for Client interface from package docker

### PR DESCRIPTION
**Pull request by zhyon404**
fix https://github.com/tensorchord/envd/issues/478

### CODE REVIEW

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Comment by gaocegege** [comment]
This PR fixes #478

**Review by gaocegege — APPROVED** [review]
LGTM :+1:
/lgtm

File: pkg/docker/docker.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ type Client interface {
 	StartBuildkitd(ctx context.Context, tag, name, mirror string) (string, error)
 
 	IsRunning(ctx context.Context, name string) (bool, error)
-	IsCreated(ctx context.Context, name string) (bool, error)
```
Comment (hezhizhen) [inline pkg/docker/docker.go:62]: It's better to add a new method `Exists` and mark `IsCreated` deprecated.

Comment (gaocegege) [reply pkg/docker/docker.go:62]: We are still in alpha, thus I think it also works although we remove it directly.

There is no project that relies on us, currently.

**Pull request merged**